docs/content/latest/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -47,9 +47,57 @@ Use the `CREATE INDEX` statement to create an index on the specified columns of
 
 `CONCURRENTLY`, `USING method`, `COLLATE`, and `TABLESPACE` options are not yet supported.
 
+When an index is created on a populated table, YugabyteDB automatically
+backfills the existing data into the index.  In most cases, this uses an online
+schema migration.
+
+| | online | not online |
+| --- | --- | --- |
+| safe to do other DMLs during `CREATE INDEX`? | yes\* | no |
+| keeps other transactions alive during `CREATE INDEX`? | mostly | no |
+| parallelizes index loading? | yes | no |
+
+To disable online schema migration for YSQL `CREATE INDEX`, set the flag
+`ysql_disable_index_backfill=true` on **all** nodes and **both** master and
+tserver.  To disable online schema migration for one `CREATE INDEX`, use
+`CREATE INDEX NONCONCURRENTLY`.
+
+If online `CREATE INDEX` fails, it likely failed in the backfill step.  In that
+case, the index exists but is not usable.  Drop the index and try again.  If it
+still doesn't work,
+
+- did it time out?  Then, try increasing timeout flags:
+  - master `ysql_index_backfill_rpc_timeout_ms`
+  - tserver `ysql_wait_until_index_permissions_timeout_ms`
+- did you get "backfill failed to connect to DB"?  Then, you may be hitting an
+  issue with authentication.  If the version is <2.4 (stable) or <2.5.2
+  (latest), online `CREATE INDEX` does not work with authentication enabled.
+  - If the version is 2.5.1 (latest), then you can use `CREATE INDEX
```

Comment:
Current latest is 2.5.3.1...
```suggestion
  - If the version is at least 2.5.1, then you can use `CREATE INDEX
```

Reply:
"(latest)" and "(stable)" emphasize the release lines since there are two.  Odds are under the latest line; evens are under the stable line.

Reply:
My point here is that you're calling out a non-current version in the `latest` release line. Are you meaning to call out 2.5.1 specifically, or just "if you're using 2.5.x, you can use CREATE INDEX"?

Line 73, as well, is out of date as of yesterday's release. This is the trouble with calling out version numbers. :(

Reply:
Updated the suggestion :)